### ISSUE TITLE
Can't switch between TUFX profiles

### ISSUE THREAD

**Issue by AdrianDogmeat** [shadowmage45/TUFX#50]
All the profiles are present in the menu, but I can't switch between them

**Comment by AdrianDogmeat** [comment]
> At the very least I'd need a ksp.log to do anything with this.

Here:
[log file.zip](https://github.com/shadowmage45/TUFX/files/14194600/log.file.zip)

**Comment by JonnyOThan** [comment]
You have a fatal B9PartSwitch error that you ignored.

**Comment by AdrianDogmeat** [comment]
To be fair this error wasn't doing much to how TUFX worked on earlier
versions
Besides, I had this for 4 years now

**Comment by JonnyOThan** [comment]
No, *fatal* errors are *fatal*.  You're not supposed to continue playing because things *will* break.  Serious warnings can usually be skipped past, but some things probably won't work as intended.

**Comment by AdrianDogmeat** [comment]
Yeah I'll probably take a look and see what I can find
